### PR TITLE
Update SQL module release notes

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: release-notes/swan-lake-beta3-release-note.md [hunk 1]
```diff
@@ -594,6 +594,24 @@ record{} queryResult = sqlClient->queryRow(`SELECT * FROM ExTable where row_id =
 int count = sqlClient->queryRow(`SELECT COUNT(*) FROM ExTable`);
 ```
 
+- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to create a dynamic/constant complex query.
+  - The `queryConcat()` creates a parameterized query by concatenating a set of parameterized queries.
+    ```ballerina
+    int id = 10;
+    int age = 12;
```
Comment (daneshk) [inline release-notes/swan-lake-beta3-release-note.md:601]:
```suggestion
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 2]
```diff
@@ -594,6 +594,24 @@ record{} queryResult = sqlClient->queryRow(`SELECT * FROM ExTable where row_id =
 int count = sqlClient->queryRow(`SELECT COUNT(*) FROM ExTable`);
 ```
 
+- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to create a dynamic/constant complex query.
```
Comment (daneshk) [inline release-notes/swan-lake-beta3-release-note.md:597]:
```suggestion
- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to dynamically create a complex query.
```

File: release-notes/swan-lake-beta3-release-note.md (hunk 1 above)
Comment (daneshk) [reply release-notes/swan-lake-beta3-release-note.md:601]: As we need to show how two queries append, we can skip these variable

**Review by daneshk — APPROVED** [review]

File: release-notes/swan-lake-beta3-release-note.md [hunk 3]
```diff
@@ -594,6 +594,20 @@ record{} queryResult = sqlClient->queryRow(`SELECT * FROM ExTable where row_id =
 int count = sqlClient->queryRow(`SELECT COUNT(*) FROM ExTable`);
 ```
 
+- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to dynamically create a complex query.
```
Comment (praneesha) [inline release-notes/swan-lake-beta3-release-note.md:597]:
```suggestion
- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to create a complex query dynamically.
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 4]
```diff
@@ -594,6 +594,20 @@ record{} queryResult = sqlClient->queryRow(`SELECT * FROM ExTable where row_id =
...
+  - The `queryConcat()` creates a parameterized query by concatenating a set of parameterized queries.
```
Comment (praneesha) [inline release-notes/swan-lake-beta3-release-note.md:598]:
```suggestion
  - The `queryConcat()` function creates a parameterized query by concatenating a set of parameterized queries.
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 5]
```diff
@@ -594,6 +594,20 @@ record{} queryResult = sqlClient->queryRow(`SELECT * FROM ExTable where row_id =
...
+    ```ballerina
+    sql:ParameterizedQuery query = `SELECT * FROM students`;
+    sql:ParameterizedQuery query1 = ` WHERE id < ${id} AND age > ${age}`;
+    sql:ParameterizedQuery sqlQuery = sql:queryConcat(query, query1);
+    ```
+  - The `arrayFlattenQuery()` is introduced to make the array flatten easier. It makes the inclusion of
```
Comment (praneesha) [inline release-notes/swan-lake-beta3-release-note.md:604]:
```suggestion
  - The `arrayFlattenQuery()` is introduced to make the array flattening easier. It makes the inclusion of 
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 6]
```diff
@@ -594,6 +594,20 @@ record{} queryResult = sqlClient->queryRow(`SELECT * FROM ExTable where row_id =
...
+    ```
+  - The `arrayFlattenQuery()` is introduced to make the array flatten easier. It makes the inclusion of 
+    varying array elements into the query easier by flattening the array to return a parameterized query.
+    ```ballerina
+    int[] ids = [1, 2];
+    sql:ParameterizedQuery sqlQuery = sql:queryConcat(`SELECT * FROM DataTable WHERE id IN (`, sql:arrayFlattenQuery(ids), `)`);
+    ```
+
 ##### WebSocket Package
 - Added OAuth2 JWT bearer grant type support for client
```
Comment (praneesha) [inline release-notes/swan-lake-beta3-release-note.md:612]:
```suggestion
- Added OAuth2 JWT bearer grant type support for the client
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**